### ISSUE TITLE
AppImage does not work

### ISSUE THREAD

**Issue by probonopd** [irccloud/irccloud-desktop#55]
```
me@host:~$ '/home/me/Downloads/irccloud-desktop-0.2.0.AppImage' 
/tmp/.mount_ti507O
installed: X-AppImage-Version=0.2.0.166 image: X-AppImage-Version=0.2.0.166
/tmp/.mount_ti507O/AppRun: line 46: /tmp/.mount_ti507O/usr/bin/irccloud: No such file or directory
```

Might be related to [this](https://github.com/electron-userland/electron-builder/issues/504#issuecomment-235372662).

**Comment by jwheare** [comment]
Thanks, hopefully fixed automatically when we upgrade electron-builder in the next release.

**Comment by probonopd** [comment]
Should be fixed in electron-builder, https://github.com/electron-userland/electron-builder/issues/504#issuecomment-237814035

**Comment by jwheare** [comment]
@probonopd can you please test this version: https://filebin.net/5ae9j8xrykvv222p/irccloud-desktop-0.3.0-x86_64.AppImage

**Comment by probonopd** [comment]
Runs, but the "Set Default" button does not work for me.

**Comment by jwheare** [comment]
Ah yes, that feature only works on Mac/Windows. I'll suppress it for Linux. Does it crash, prevent the app from continuing to load or just do nothing?

**Comment by probonopd** [comment]
It just causes the dialog to re-appar again forever...

**Comment by jwheare** [comment]
Ah yes, I can see how it would. Will send over a new build in a moment.

**Comment by jwheare** [comment]
OK, this one should work without the dialog https://filebin.net/swo8cnig1mkkwkfy/irccloud-desktop-0.3.0-x86_64.AppImage

**Comment by probonopd** [comment]
Works, thanks.

**Comment by jwheare** [comment]
This fix is now released in v0.3.0

**Comment by jwheare** [comment]
I meant to say: thanks for testing!
